deployments/helm/values.yaml

```diff
@@ -0,0 +1,157 @@
+image:
+  repository: grafana/beyla
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
Rather than default to latest, I suggest leaving this blank, but then default to the Chart's AppVersion. That way we're never defaulting to latest.
Here's how the Grafana Agent does it:
values.yaml: https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/values.yaml#L99-L112
image spec in the container: https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/templates/containers/_agent.yaml#L3
_helpers.tpl: https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/templates/_helpers.tpl#L85-L100